amr-wind/utilities/diagnostics.cpp

```diff
@@ -1,8 +1,148 @@
 #include "amr-wind/incflo.H"
 #include "diagnostics.H"
+#include "constants.H"
 
 using namespace amrex;
 
+void amr_wind::diagnostics::get_mfab_extrema(
+    amrex::Real& mfab_max_val,
+    amrex::Real& mfab_min_val,
+    const amrex::MultiFab& mfab,
+    const int nghost)
+{
```

Comment:
Could we use MultiFab min() and max()? Or even BaseFab::minmax()? And then for the masked version we could do a Multiply to get the mfab to do the reduction on?